free-2.md

```diff
@@ -58,12 +78,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I am not sure about this yet, but I supppose it would be 50/50 roughly. A lot of design choices have to be made, but at the same time, it will take time to implement all of the things I've mentioned and more.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+It is fairly moderately sized. It is not too ambitious as it just focuses on making one kind of chart, but may require a bit of work.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+It is a good idea for the design choices one has to make in making Gantt chart creation intuitive. However, it might not also be a good idea because it may be the case that using cells may in fact be the most intuitive way to build Gantt charts. In addition, people may believe that the time to learn a DSL in making Gantt charts may not be as useful as the time to learn a generic suite for projects.
```

Comment:
This last two lines really sum up the trade-offs of DSL's; whether the initial investment of time to learn the DSL is worth it for the improved productivity. In this case, there's an additional factor to consider: whether this DSL would be more effective than existing tools (that are more like gui's I imagine?). This sounds like one of those things where it would be very helpful to run through ideas/interview several different people who use Gantt charts extensively to get their take on it.